packages/web3-providers/src/providers/Web3EthereumProvider.js

```diff
@@ -129,14 +128,18 @@ export default class Web3EthereumProvider extends AbstractSocketProvider {
      * @returns {Promise<Object>}
      */
     async send(method, parameters) {
-        const response = await this.connection.send(method, parameters);
-        const validationResult = JsonRpcResponseValidator.validate(response);
+        let result;
+        try {
+            result = await this.connection.send(method, parameters);
+        } catch (error) {
+            throw new Error(`Node error: ${error.message}`);
+        }
 
-        if (validationResult instanceof Error) {
-            throw validationResult;
+        if (result === undefined) {
```

Comment:
This check isn't required :)

Reply:
fix: https://github.com/ethereum/web3.js/pull/2862/commits/747554bce786962eae0f04db867d2365cd153551